docs/configuration.md

```diff
@@ -171,6 +171,21 @@ git config --global credential.gitHubAuthModes "oauth,basic"
 
 ---
 
+### vstsScope
+
+Overrides GCM default scope request when generating a Personal Access Token from Azure DevOps.
+The supported format is one or more [scope values](https://docs.microsoft.com/en-us/vsts/integrate/get-started/authentication/oauth#scopes) separated by whitespace, commas, semi-colons, or pipe `'|'` characters.
```

Comment:
New URL, and lets reduce the set of value separators to just commas (pipes and semi-colons might be interpreted by the particular shell flavour the user is using).

```suggestion
The supported format is one or more [scope values](https://docs.microsoft.com/en-us/azure/devops/integrate/get-started/authentication/oauth) separated by commas (`,`).
```